packages/cli/src/lib/commands/collect/options/dryRun.ts

```diff
@@ -1,18 +1,14 @@
-import { argv } from 'yargs';
-import { Param } from './dryRun.model';
-import { ArgvOption } from '../../../core/yargs/types';
+import { argv, Options } from 'yargs';
 import { getEnvPreset } from '../../../pre-set';
 
-export const param: Param = {
-  dryRun: {
-    alias: 'd',
-    type: 'boolean',
-    description: 'Execute commands without effects',
-    default: getEnvPreset().dryRun as boolean
-  }
-};
+export const dryRun = {
+  alias: 'd',
+  type: 'boolean',
+  description: 'Execute commands without effects',
+  default: getEnvPreset().dryRun as boolean
+} satisfies Options;
 
 export function get(): boolean {
-  const { dryRun } = argv as any as ArgvOption<Param>;
+  const { dryRun } = argv as any as { dryRun: boolean };
   return dryRun;
 }
```

Comment:
It cannot be removed because the function is used. If breaks the flow of the application and i believe because of the changes with the way node handles bindings it will not work in ESM

Reply:
Unfortunatly this requires a much larger change and maybe belongs in a seperate MR